servers/physics_3d/area_3d_sw.h

```diff
@@ -232,14 +231,13 @@ void Area3DSW::remove_area_from_query(Area3DSW *p_area, uint32_t p_area_shape, u
 }
 
 struct AreaCMP {
-	Area3DSW *area;
-	int refCount;
+	Area3DSW *area = nullptr;
+	int refCount = 1;
```

Comment:
That's not consistent with the change done in Area2D, which are IMO more correct.

There are two constructors, and one doesn't set `area`, so the default `refCount` should be 0.